include/support.h

```diff
@@ -122,4 +122,9 @@ void upcase(std::string &str);
 void lowcase(std::string &str);
 void strip_punctuation(std::string &str);
 
+template<size_t N>
+bool starts_with(const char (& pfx)[N], const std::string &str) noexcept {
+	return (strncmp(pfx, str.c_str(), N) == 0);
+}
+
```

Comment:
Another nice addition!

Reply:
I just miss Python's sensibly named functions for operating on strings :) I considered adding `bool starts_with(const std::string &, const std::string&)` as well, but that function would be left unused, so this variant is enough for now.